ide/csl.api/src/org/netbeans/modules/csl/editor/overridden/ComputeAnnotations.java

```diff
@@ -127,6 +127,10 @@ public void run(Result result, SchedulerEvent event) {
 
                                     while (!todo.isEmpty()) {
                                         StructureItem i = todo.remove(0);
+                                        if (i instanceof StructureItem.InheritedItem
+                                                && ((StructureItem.InheritedItem) i).isInherited()) {
```

Comment:
Rather than spreading these instance checks all over the place, there could be a helper method:
```java
public static boolean isInherited(StructureItem i) {
}
```
it could actually be in `StructureItem` interface itself now, when it is possible to have static methods in interfaces.

Reply:
Thank you for your review! Will do that.

Reply:
Fixed: https://github.com/apache/netbeans/compare/28c4692734c21860b277316982c0fd97988b6c3e..413de5540ab999605599cc876989b4fc97f2b5e7